### PR TITLE
[FEATURE] TimeNode Wallet Improvements

### PR DESCRIPTION
**Pull request by Bagaric**
Asks the user for the wallet password every time he opens the TimeNode screen to enable the TimeNode.

Please squash commits if merging.

### CODE REVIEW

File: app/components/Header/Header.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Header extends Component {
 
   render() {
     const { web3Service } = this.props;
-    const networkNameString = web3Service.network ? web3Service.network.name : '';
+    const networkNameString = web3Service.network ? web3Service.network.name : 'Unknown';
```
Comment (Kuzirashi) [inline app/components/Header/Header.js]: This requires updating snapshot:

`yarn test -u` or `jest -u`

File: app/components/TimeNode/TimeNodeUnlock.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import PropTypes from 'prop-types';
+import { inject, observer } from 'mobx-react';
+import PoweredByEAC from '../Common/PoweredByEAC';
+
+@inject('timeNodeStore')
+@observer
+class TimeNodeUnlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (Kuzirashi) [inline app/components/TimeNode/TimeNodeUnlock.js]: No need for using state in this component. Actually, you're not using `this.state.updateWalletUnlocked` anywhere, you use `this.props.updateWalletUnlocked` on line 36, which is good.

File: app/components/TimeNode/TimeNodeWallet.js [hunk 3]
```diff
@@ -66,23 +61,23 @@ class TimeNodeWallet extends Component {
...
-                type="password"
-                placeholder="Password"
-                className="form-control"
-                ref={(el) => this.passwdRef = el}/>
-            </div>
-          </div>
-        </div>
-      </div>;
-    }
+    // let passwordField = null;
```
Comment (Kuzirashi) [inline app/components/TimeNode/TimeNodeWallet.js]: Commented out code?

File: app/components/TimeNode/TimeNodeUnlock.js (hunk 2 above)
Comment (Bagaric) [reply app/components/TimeNode/TimeNodeUnlock.js]: Hahah yeah leftovers.

**Pull request merged**